### PR TITLE
Transfer tests to IntrevalArithmetic.jl

### PR DESCRIPTION
**Pull request by krish8484**

### CODE REVIEW

File: transfer_tests.jl [hunk 1]
```diff
@@ -0,0 +1,16 @@
+using IntervalArithmetic
+using Base.Filesystem
+
+output_file_path = joinpath(dirname(dirname(pathof(IntervalArithmetic))), "test", "ITF1788_tests")
+input_file_path = joinpath(@__DIR__, "output", "julia", "Base.Test", "ValidatedNumerics")
+files = readdir(relpath(input_file_path, @__DIR__))
+
+for file in files
```
Comment (dpsanders) [inline transfer_tests.jl:9]: Probably better to use the `Glob.jl` package to just choose the `.jl` files.

**Comment by dpsanders** [comment]
I'm not sure what's going on with Travis?

File: transfer_tests.jl [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 
 for file in files
-    if(file=="Makefile")
-        continue
-    end
+    bname = basename(file)
     filepath = joinpath(input_file_path, file)
-    ofilepath = joinpath(output_file_path, file)
+    ofilepath = joinpath(output_file_path, bname)
     cp(relpath(filepath, @__DIR__), ofilepath; force=true)
```
Comment (dpsanders) [inline transfer_tests.jl]: Do you really need `@__DIR__` here?  
(You might, it just feels wrong.)

Comment (krish8484) [reply transfer_tests.jl]: We don't, I was just generalizing, changes have been made to remove it.

**Pull request merged**